trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -311,6 +311,30 @@ const sendFirstTaskEmail = async (taskVariables) => {
   }
 };
 
+/**
+ * Initiate an internal UKEF team email, upon an amendment
+ * submission (automatic) and approval (manual)
+ * @param {String} ukefFacilityId UKEF Facility ID
+ */
+const internalAmendmentEmail = async (ukefFacilityId) => {
+  try {
+    const templateId = EMAIL_TEMPLATE_IDS.INTERNAL_AMENDMENT_NOTIFICATION;
+    const sendToEmailAddress = process.env.UKEF_INTERNAL_NOTIFICATION;
+    const emailVariables = {
+      ukefFacilityId,
+    };
+
+    if (!templateId || !sendToEmailAddress || !ukefFacilityId) {
+      throw new Error('Void argument specified');
+    }
+
+    return sendTfmEmail(templateId, sendToEmailAddress, emailVariables);
+  } catch (error) {
+    console.error('Error sending internal amendment email', { error });
+    return false;
```

Comment:
can we return a status/message instead? `false` may not be descriptive

Reply:
Good shout, have updated accordingly.